### PR TITLE
V0.14.0 rc

### PR DESCRIPTION
**Pull request by rhornung67**
# Summary

- This PR is the release candidate branch for the v0.14.0 release.
- It contains new release notes, bumps version numbers, etc.

### CODE REVIEW

**Comment by artv3** [comment]
PR #1029  is very close, I noticed the kernels are not running async though. They are on different streams but not async. I was wondering if @mdavis36 @trws might have some insight. I'm not sure where the synchronization points are coming from.

Could we add PR #1078   to the list?

**Review by MrBurmark — DISMISSED** [review]

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -10,6 +9,133 @@
...
+        involving OpenMP scheduling are consistent. Specification of a chunk
+        size for scheduling policies is optional, which is consistent with
+        native OpenMP usage. In addition, no-wait policies are more constrained
+        to prevent potentially non-conforming (to the OpenMP spec) usage. 
+        Finally, additional policy type aliases have been added to make common 
+        use cases less verbose. Please see the RAJA policy documentation in 
+        the User Guide for policy descriptions. 
+      * Host implementation of Hip atomics added.
+      * Add ability to specify atomic to use on the host in CUDA and Hip
+        atomic policies (i.e., added host atomic templlate parameter), This
```
Comment (MrBurmark) [inline RELEASE_NOTES.md:60]: typo

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -10,6 +9,133 @@
...
+        is useful for host-device deorated lambda expressions that may be
```
Comment (MrBurmark) [inline RELEASE_NOTES.md:61]: typo

File: RELEASE_NOTES.md (hunk 1 above)
Comment (rhornung67) [reply RELEASE_NOTES.md]: Fixed

File: RELEASE_NOTES.md (hunk 2 above)
Comment (rhornung67) [reply RELEASE_NOTES.md]: fixed

**Comment by rhornung67** [comment]
@MrBurmark I fixed the two typos you found and found two others. The v0.14.0 release notes section should be good now. Thanks.

**Review by MrBurmark — APPROVED** [review]

**Review by artv3 — APPROVED** [review]

**Pull request closed without merging**